### PR TITLE
stage_as

### PR DESCRIPTION
**Pull request by VChapdelaineT**
avoid file_name collision

### CODE REVIEW

File: WES+WGS/Pipeline.nf [hunk 1]
```diff
@@ -11,14 +11,13 @@
...
-   memory "4 GB"
+   memory "1 GB"
    time "2h"
    errorStrategy 'retry'
    maxRetries 1
    
    input:
-   tuple val(sample), path(wes), path(wgs)
-   
+      tuple val(sample), path(wes, stageAs: 'WES.bam'), path(wgs, stageAs: 'WGS.bam')
```
Comment (dtaliun) [inline WES+WGS/Pipeline.nf:20]: OK

File: WES+WGS/Pipeline.nf [hunk 2]
```diff
@@ -11,14 +11,13 @@
 process HarmonizeSM {
    cache "lenient"
    cpus 1
-   memory "4 GB"
+   memory "1 GB"
```
Comment (dtaliun) [inline WES+WGS/Pipeline.nf:14]: This applies only for your situation and should not be recommended as default

File: WES+WGS/Pipeline.nf [hunk 3]
```diff
@@ -30,16 +29,16 @@ process HarmonizeSM {
...
+   
    cpus 1
-   memory "16 GB"
-   time "6h"
+   memory "1 GB"   
+   time '2h'
    errorStrategy 'retry'
-   maxRetries 1
-   
+   maxRetries 3
```
Comment (dtaliun) [inline WES+WGS/Pipeline.nf:38]: Doesn't make sense to have maxRetries 3.... Values apply only to your situation

File: WES+WGS/Pipeline.nf [hunk 4]
```diff
@@ -30,16 +29,16 @@ process HarmonizeSM {
...
 
    input:
-   tuple val(sample), path(wes), file(wgs)
+   tuple val(sample), path(wes), path(wgs)
```
Comment (dtaliun) [inline WES+WGS/Pipeline.nf:41]: OK

**Review by dtaliun — CHANGES_REQUESTED** [review]
In the commit comment, you said that you added only stage_as, but you also did many changes in the default settings. These should have been two different commits or must be reflected in the comments. In any way, we need only fixes with the "stage_as" and "path".

File: WES+WGS/Pipeline.nf (hunk 2 above)
Comment (VChapdelaineT) [reply WES+WGS/Pipeline.nf]: reverted

File: WES+WGS/Pipeline.nf (hunk 3 above)
Comment (VChapdelaineT) [reply WES+WGS/Pipeline.nf]: reverted

**Review by dtaliun — APPROVED** [review]

**Pull request merged**